api-server/README.md

```diff
@@ -217,7 +217,7 @@ The `-h` or `--help` option can list the available options of the `llama-api-ser
     -a, --model-alias <MODEL-ALIAS>
             Sets the alias name of the model in WasmEdge runtime [default: default]
     -c, --ctx-size <CTX_SIZE>
-            Sets the prompt context size [default: 4096]
+            Sets the prompt context size [default: 512]
```

Comment:
If you reduce the `ctx-size`, the `batch-size` should also reduce.

Reply:
What is an appropriate value for `batch-size`? also `512`?

Reply:
Let's use the default value as the llama.cpp did: https://github.com/ggerganov/llama.cpp/blob/master/common/common.h#L51-L52

`512`.

Reply:
Accepted.